packages/common/src/types/config.ts

```diff
@@ -1,65 +1,62 @@
 import type { ElasticSearchConfig } from './clientConfig.js';
 import type { LoggerConfig } from './logger.js';
-
-type BindingConfig = {
-	topic: string;
+interface BindingConfig {
 	dlq: string;
-};
-
-type SchemaBindingConfig = {
+	topic: string;
+}
+interface SchemaBindingConfig {
 	analysisMessage: BindingConfig;
 	requestMessage: BindingConfig;
-};
-
-export type KafkaConfig = {
-	lyricSchemaBinding?: SchemaBindingConfig;
+}
+export interface KafkaConfig {
 	enabled: boolean;
+	lyricSchemaBinding?: SchemaBindingConfig;
 	servers?: string;
 	songSchemaBinding?: SchemaBindingConfig;
-};
+}
+
+export const RepositoryType = {
+	SONG: 'SONG',
+	LYRIC: 'LYRIC',
+} as const;
 
-type SongRepositoryConfig = {
+type ValueOf<T> = T[keyof T];
```

Comment:
this is a common utility type that you could export from its own file for global use. Typically I call this `Values<T>`.